src/main/scala/logger/SymbExLogger.scala

```diff
@@ -545,22 +448,143 @@ class SymbLog(v: ast.Member, s: State, pcs: PathConditionStack) {
     }
   }
 
+  /**
+    * Calculates the diff between the current and last statistics query.
+    * The difference is calculated if value can be converted to an int or double
+    *
+    * @return map with the current statistics, and the differences (only containing values that could be converted)
+    *         and keys with appended "-delta"
+    */
+  def deltaStatistics(currentStatistics: Map[String, String]): Map[String, String] = {
+    val deltaStatistics = currentStatistics map getDelta filter { case (_, value) => value.nonEmpty } map {
+      case (key, Some(value)) => (key + "-delta", value)
+      case other => sys.error(s"Unexpected result pair $other")
+    }
+    whenEnabled { lastStatistics = lastStatistics ++ currentStatistics }
+    currentStatistics ++ deltaStatistics
+  }
+
+  private def getDelta(pair: (String, String)): (String, Option[String]) = {
+    val curValInt = pair._2.toIntOption
+    val prevValInt = lastStatistics.get(pair._1) match {
+      case Some(value) => value.toIntOption
+      case _ => Some(0) // value not found
```

Comment:
Magic constants are dangerous. Can't `None` be returned instead?

Reply:
Yes, the `-delta` key will be missing from the first statistics, but that makes sense.